backend/node/genvm/base.py

```
@@ -31,7 +33,9 @@
 @dataclass
 class ExecutionError:
     message: str
-    kind: typing.Literal[ResultCode.USER_ERROR, ResultCode.VM_ERROR]
+    kind: typing.Literal[
+        ResultCode.USER_ERROR, ResultCode.VM_ERROR, ResultCode.INTERNAL_ERROR
```

Ruff (0.12.2)

`ResultCode` may be undefined, or defined from star imports

(F405)